utils/libtrixi-init-julia

```diff
@@ -314,17 +314,24 @@ echo
 
 # Set system HDF5 library
 echo "Use system HDF5 for Julia... "
+# Disable preloading libcurl.so if it has been set when running within the GitHub runner,
+# since it does not work with the JLL-provided libhdf5.so
+if [ "$CI" = "true" ] && [ -n "$LD_PRELOAD" ]; then
+  store_ld_preload="$LD_PRELOAD"
+  LD_PRELOAD=""
+fi
 JULIA_HDF5_PATH="$hdf5_libdir" JULIA_DEPOT_PATH="$julia_depot" $julia_exec --project=. \
```

Comment:
```suggestion
JULIA_DEPOT_PATH="$julia_depot" $julia_exec --project=. \
```
With the preferences setup working, setting this variable should not be necessary anymore